ndsl/__init__.py

```diff
@@ -1 +1,62 @@
+from .buffer import Buffer
+from .comm.boundary import Boundary, SimpleBoundary
+from .comm.communicator import CubedSphereCommunicator, TileCommunicator
+from .comm.local_comm import AsyncResult, ConcurrencyError, LocalComm
+from .comm.mpi import MPIComm
+from .comm.null_comm import NullAsyncResult, NullComm
```

Comment:
Dunno that NullAsyncResult needs to be exposed here

Reply:
Same thing, `typing`